CHANGELOG.md

```diff
@@ -1,11 +1,52 @@
 # Changelog
 
-**THIS CHANGELOG HAS BEEN DEPRECATED**
+This changelog contains notable changes that make into each [`cardano-node` release](https://github.com/input-output-hk/cardano-node/releases),
```

Comment:
This is perfectly  understandable, but maybe "changes that made it into"  or "that make it into" would sound a little more neat?

Reply:
Funny that you correct me there, cause it was `made`, but I changed it to `make` :smile: 

Thank you for correcting me back! Fixed.